packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -155,8 +155,9 @@ class Ticker {
     }());
     assert(_startTime == null);
     _future = new TickerFuture._();
-    if (shouldScheduleTick)
-      scheduleTick();
+    if (!muted && !scheduled) {
```

Comment:
Looking at this again I don't think we need special logic for start, but like we discussed yesterday -
We shouldn't care about whether or not we are idle when starting the ticker, since it won't tick anyway if we are paused.  Instead we can simplify this by moving the first two isTicking conditions into `shouldScheduleTick` and removing the debug assert for `isTicking` in start.